SOP/SOP-push.adoc

```diff
@@ -0,0 +1,179 @@
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:table-caption!:
+endif::[]
+
+:toc:
+:toc-placement!:
+
+= UnifiedPush Operator - Standard Operating Procedures
+
+:toc:
+toc::[]
+
+== Critical
+
+=== UnifiedpushDown or UnifiedPushConsoleDown
+
+Troubleshoot using the following steps via either the console or the cli:
+
+==== Console:
+
+. Check that the UnifiedPushServer CR is deployed in the same namespace as the operator
+. Check the environment variables of the Server
+.. Go to : `Applications -> Pods -> example-unifiedpushserver-<xyz123> -> Environment Variables`
+.. For further information see https://github.com/aerogear/aerogear-unifiedpush-server#container-configuration.
+.. Note that it will use the values mapped in the Secret created by the operator with the database pod name.
+. Check the environment variables of the Database
+.. Go to : `Applications -> Pods -> example-unifiedpushserver-postgresql-<xyz123> -> Environment Variables`
+.. By default it should use the values defined in the Secret created by the operator with the database pod name.
+. Check the logs of the UPS OAuth Proxy Container
+.. Go to : `Applications -> Pods -> example-unifiedpushserver-<xyz123> -> Logs -> Container -> ups-oauth-proxy`
+.. Check the `pod/example-unifiedpushserver-<xyz123>` events, see if was possible to pull the OAuth image.
+.. Check the logs of the UPS Container
+.. Go to : `Applications -> Pods -> example-unifiedpushserver-<xyz123> -> Logs -> Container -> ups`
+. Check the `pod/example-unifiedpushserver-<xyz123>` events, see if was possible to pull the UnifiedPush image.
+. Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+. Check if the secret was created (It cannot be checked by the UI)
+. Check if the values in the secret are correct (It cannot be checked by the UI)
+
+NOTE: If resolving the UnifiedPushOperatorDown doesn't resolve the issue, please continue with the below steps
+
+==== CLI
```

Comment:
is it really a requirement to have instructions using both `console` and `cli`? can we just stick with one of them (preferably CLI)?

Reply:
It is the way that @laurafitzgerald did the MSS one. 
@laurafitzgerald why you added the both?

Reply:
I would suggest using the CLI only, since the UI Console will change significantly in OpenShift 4+

Reply:
I do not think that these steps changed in OCP 4 at all. However, I am following as it was done and approved in other places which show attend the requirements. I think we can/should move with both them check it with @laurafitzgerald in order to know if we can keep just the CLI which indeed I am very OK with.